src/Adapters/DatabaseApproverRepository.php

```diff
@@ -75,7 +76,15 @@ public function setApproverCategories( int $userId, array $categoryNames ): void
 	}
 
 	private function serializeCategories( array $categories ): string {
-		return implode( '|', $categories );
+		return implode( '|', array_unique( array_map(
```

Comment:
Need to use only unique values, otherwise it ends up saving `Foo` twice when the array contains `foo` and `Foo`.